akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -390,6 +390,15 @@ class Source[+Out, +Mat](delegate: scaladsl.Source[Out, Mat]) extends Graph[Sour
     new Source(delegate.scan(zero)(f.apply))
 
   /**
+   * Similar to `scan` but only emits the current value once, when completing.
+   * Its current value which starts at `zero` and then
+   * applies the current and next value to the given function `f`,
+   * yielding the next current value.
```

Comment:
Ah, it seems that we forgot to synchronize the doc comments between Flow and Source for Java: we should add the supervision and semantics documentation sections for all operations here as well.